core/src/scss/components/date-stacked/_date-stacked.scss

```diff
@@ -0,0 +1,28 @@
+@charset "UTF-8";
+
+//
+// Date Stacked
+//
+// Abbreviated month stacked on top of numerical day of the month.
+//
+// .su-date-stacked--no-background - Black type on transparent background
+//
+// Markup: ../templates/components/date-stacked/date-stacked.twig
+//
+// Style guide: Components.Date Stacked
+//
+.su-date-stacked {
+  @include modular-spacing('padding', 0);
+  background: $color-black;
+  color: $color-white;
+  text-align: center;
+
+  .su-date-month {
```

Comment:
I think the BEM naming conventions suggest that this should be named `.su-date-stacked__month`. For brevity, I might be ok with `.su-date__month`. Depends on if we're ever going to have a date that isn't stacked.

Reply:
You're right! This is an oversight by me. Thanks for catching this.